content/en/legacy/helm-operator/get-started/quickstart.md

```diff
@@ -288,4 +288,4 @@ sophisticated setup options:
 
 Want to take a deeper dive in the available features and the
 `HelmRelease` resource?
-**Continue with the [`HelmRelease` guide](../helmrelease-guide/introduction.md).**
\ No newline at end of file
+**Continue with the [`HelmRelease` guide](../helmrelease-guide/introduction.md).**
```

Comment:
What is this change about?

Reply:
@dholbach It appears that GitHub automatically added a newline at the end of this Markdown document.